### PR TITLE
Added `coverage` reporter

### PR DESCRIPTION
**Pull request by sprilukin**
Added `coverage` reporter when karma started in coverage mode, otherwise plugin is asking for lcov file

### CODE REVIEW

**Comment by sprilukin** [comment]
I signed it!

File: lib/intellijCoverageReporter.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ function configureAngularCliCoverage(config) {
 function configureCoverage(config) {
   var reporters = config.reporters || [];
   if (cli.isWithCoverage()) {
+    reporters.push('coverage');
```
Comment (segrey) [inline lib/intellijCoverageReporter.js:36]: Maybe `reporters.push('coverage')` could be added if `karma-coverage` plugin is used. Looks good to me.
This repo is outdated, let's close this one and continue in the https://youtrack.jetbrains.com/issue/WEB-32836

**Comment by segrey** [comment]
@sprilukin Please close this PR, it will be fixed in IntelliJ/WebStorm 2021.1.

**Comment by sprilukin** [comment]
closed per request @segrey

**Pull request closed without merging**